flowdb/Dockerfile

```diff
@@ -23,8 +23,7 @@ ENV POSTGIS_MAJOR=$POSTGIS_MAJOR
 ARG POSTGIS_VERSION=2.5.2
 ENV POSTGIS_VERSION=$POSTGIS_VERSION
 ENV POSTGRES_DB=flowdb
-ARG POSTGRES_USER=flowdb
-ENV POSTGRES_USER=$POSTGRES_USER
+ENV POSTGRES_USER=$FLOWDB_ADMIN_USER
```

Comment:
I think you need to declare `FLOWDB_ADMIN_USER` as either an ARG or an ENV before using it here.